manager/manager.py

```diff
@@ -1,15 +1,53 @@
+import os
 import json
 import select
 import subprocess
 from datetime import datetime
 from typing import Optional
+from status import Status
+
+base_dir = "./ignore/opt/orbs"
+os.makedirs(f"{base_dir}/manager", exist_ok=True)
+errors_file = f"{base_dir}/manager/errors.txt"
+status_file = f"{base_dir}/manager/status.json"
+log_file = f"{base_dir}/manager/log.txt"
 
-errors_file = '/opt/orbs/errors.txt'
-node_version = '/opt/orbs/node-version.json'
 timestamp = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
 
+# status object
+status = Status()
+
+data = {
+    "currentVersion": "0.0.0",
+    "scheduledVersion": None,
+    "updateScheduled": None,
+    "updateScheduledFor": None,
+}
+
+
+# logger
+def append_line_to(file_name, line):
```

Comment:
Possible to add type args (see what I did for `run_command` for an example)?

Reply:
sure thing